komodo-relational/src/main/java/org/komodo/relational/dataservice/internal/DataserviceImpl.java

```diff
@@ -323,6 +323,20 @@ public void setDescription(final UnitOfWork uow, final String newDescription) th
     }
 
     @Override
+    public String[] getVdbPlan(UnitOfWork transaction) throws KException {
```

Comment:
These TODOs point to where this class will be updated once the Dataservice Sequencer comes on stream